src/UserSpaceInstrumentation/AttachTest.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/time/clock.h>
+#include <gtest/gtest.h>
+#include <sys/types.h>
+
+#include <set>
+#include <vector>
+
+#include "OrbitBase/GetProcessIds.h"
+#include "TestProcess.h"
+#include "UserSpaceInstrumentation/Attach.h"
+
+namespace orbit_user_space_instrumentation {
+
+TEST(AttachTest, AttachAndStopProcess) {
+  TestProcess test_process;
+  const pid_t pid = test_process.pid();
+
+  const auto result = AttachAndStopProcess(pid);
+  ASSERT_TRUE(result) << result.error().message();
+
+  // TestProcess is continuously spawning new threads when it is running. Verify that no new threads
+  // get spawned - i.e. the process is not running anymore.
+  const auto tids = orbit_base::GetTidsOfProcess(pid);
+  absl::SleepFor(absl::Milliseconds(50));
```

Comment:
this is not related to this PR: I was thinking that maybe we should start moving away from absl time functions. that is if stl already has everything we need.

Reply:
Well Dimitry, I'm glad we finally agree on this. But I have to say that the `absl` version is generally easier to use, `chrono` is sometimes unintuitive. So I'm very much in favor of leaving it as preference.

Reply:
Hmm, usability in abseil is better. I always need to read up stuff before using std::chrono.